src/LogProxy.ts

```diff
@@ -35,8 +35,9 @@ export async function logMessage(level: LogLevel, module: string, message: strin
         if (level === LogLevel.WARN) clientMessage = `⚠ | ${message}`;
         if (level === LogLevel.ERROR) clientMessage = `‼ | ${message}`;
 
-        const roomIds = [config.managementRoom, ...additionalRoomIds];
         const client = config.RUNTIME.client;
+        const managementRoomId = await client.resolveRoom(config.managementRoom);
```

Comment:
this bit isn't great, but I couldn't think of something better given the logging proxy can't practically stop being static

Reply:
We will follow up on this to unblock CI now FTR